File: term2/week1/week1-c/exercise4-5.c
#include <stdio.h>

int main()
{  
    int total = 0;

    for (int i = 0; i < 101; i++)
    {
        total = total + i;
    }
    float average = (float)total / 100;
    printf("Total %i\nAverage: %.2f\n", total, average);
    
    return 0;
}

File: term2/week1/week1-rs/exercise4-5.rs
fn main() {
    let mut numbers = Vec::new();

    for i in 1..101 {
        numbers.push(i);
    }
    let sum: usize = numbers.iter().sum();
    let average = sum as f32 / numbers.len() as f32;
    println!("Sum: {}\nAverage: {}", sum, average);
}